app/services/automate_import_json_serializer_service.rb

```diff
@@ -0,0 +1,53 @@
+class AutomateImportJsonSerializerService
+  def serialize(import_file_upload)
+    File.open("automate_temporary_zip.zip", "wb") { |file| file.write(import_file_upload.binary_blob.binary) }
```

Comment:
Having a fixed name in a know location w/o tests for existence about it is not secure.

http://ruby-doc.org/stdlib-1.9.3/libdoc/tempfile/rdoc/Tempfile.html should be uses instead.